BetterMD/elements/table.py

```
@@ -118,14 +120,25 @@ def to_rst(self, inner, symbol, parent, **kwargs):
             return "".center(width)
         return f"**{content}**".center(width)
 
+class ThRST(CustomRst):
```

Ruff (0.8.2)

Redefinition of unused `ThRST` from line 115

(F811)